### DISCUSSION TITLE
1.15.0

### DISCUSSION THREAD

**Discussion by th-ch** [pear-devs/pear-desktop#533] [General]
_[Full diff](https://github.com/th-ch/youtube-music/compare/v1.14.0...v1.15.0)_

- update various dependencies - including electron major version
- Improve Lyrics plugin CSS by @MiepHD 
- Fix disable-autoplay plugin by @Araxeus 
- Improve video-toggle plugjn by @Araxeus 
- Add exponential volume plugin by @Rubecks 
- Improve blurry navbar by @Araxeus 
- Fix MPRIS in snap package by @markbaas 
- Fix Google Login by @Araxeus 
- Add album art to Discord by @MulverineX 
- switch test suite to Playwright
- Add presets in downloader plugin

Credits to all contributors who worked on this release! 🎉

<hr /><em>This discussion was created from the release <a href='https://github.com/th-ch/youtube-music/releases/tag/v1.15.0'>1.15.0</a>.</em>

**Comment by lg-nineoneninethreenine** [comment]
Anything new here @th-ch? Would love to test this!

**Reply by th-ch** [reply]
You can download the new release at https://github.com/th-ch/youtube-music/releases/tag/v1.15.0 to test it :)

**Reply by aaravdua18** [reply]
when is the final version gonna release of 1.15.0, btw great work @th-ch keep it up

**Comment by Bellisario** [comment]
Seems to crash after trying to load any song: Windows reports the application is not anymore "responding" correctly...

(After this, I reopened the previous version and it's working, so, that's a bug of the new version?)

**Reply by Araxeus** [reply]
Can you try installing via the setup if you used the portable exe?

**Reply by Bellisario** [reply]
Tried with the installed version now, same problem... just that in this case the music started and after less than a second Windows reported the app was not responding, after about 3 seconds the music stopped, too.

P.S.: there is another strange thing... after closing the app by the Windows alert not all the app tasks closes and I need to kill a process (or more sometimes) with the tasks manager.

**Reply by Araxeus** [reply]
Could you try renaming `config.json` to `config_BACKUP.json`?

If that works could you post here the file so we can understand what happened?

> config.json folder path:
> * Windows: `%AppData%/Youtube Music` (MyUser/AppData/Roaming/Youtube Music)
> * Linux: `~/.config/Youtube Music`
> * MacOS: `~/Library/Application\ Support/YouTube\ Music/config.json`

see #542

**Reply by lg-nineoneninethreenine** [reply]
@Araxeus for the Linux path, I think it's better as `~/.config/Youtube\ Music` since Linux formats spaces with a forward slash.

**Reply by Bellisario** [reply]
@Araxeus, I tested renaming the file but the problems keep going, could be I'm using Windows 7 (instead of Windows 10 or a newer Windows version)?

**Reply by Araxeus** [reply]
Truely have no idea, without any error code its hard to debug.
 If you can/want - clone the repo and launch app in dev mode from cmd using `yarn start`. That should hopefully give some indications whats going on.

Add me on discord if you need further help
`Araxeus#0819`

**Comment by PoDiax** [comment]
Getting "This browser or app may not be secure" on Windows
The setup one YouTube-Music-Setup-1.15.0.exe.
Other one works normal.
![](https://user-images.githubusercontent.com/43817108/148000889-4dafd23b-a666-439c-97f0-a0e3f5d7dee1.png)

**Reply by Araxeus** [reply]
Did you press retry and it said the same thing? :0

**Reply by Araxeus** [reply]
See #538

**Reply by PoDiax** [reply]
> Did you press retry and it said the same thing? :0

Yes.
This happens only in version that you need to install (YouTube-Music-Setup-1.15.0.exe)

**Comment by Q3ud** [comment]
I've been waiting for this so much, thanks for all the hard works!
I really(x2) love the discord feature :)

**Comment by woo-zy** [comment]
Hi there, I love the Discord app integration! I have problems with the touchbar though, it doesn't show the full album art - and the 'thumbs up' and 'thumbs down' button are mixed up.
